libarchive/archive_read_support_format_rar5.c

```diff
@@ -2567,7 +2568,6 @@ static int parse_tables(struct archive_read* a, struct rar5* rar,
 		}
```

Comment:
If we add `else` here to contain the next section, you can leave the two `n` variables where they were.  I think you can also drop the `continue` above?

Reply:
You can do that, but that's a bigger change.

As far as I see both n are set with call to read_bits_16, so no conflict reusing just one n variable.